### ISSUE TITLE
Phantom enter press before command is run?

### ISSUE THREAD

**Issue by samperlmutter** [SwarShah/hyper-custom-touchbar#11]
Whenever I run a command, the plugin seems to press enter before running it, causing a blank line before the command. Here is a screenshot of what happens after pressing the `git status` command from the example twice from an initally `clear`ed screen.

![issue](https://i.imgur.com/ef8D26j.png)

**Comment by swarshah** [comment]
Yes, that is very much intentional. It basically does `^C` and it is to make sure that the command runs fine even if you something written in terminal.

But you are right, if nothing is written then it shouldn't do that and execute command directly.

I think we can use `SESSION_ADD_DATA` or `SESSION_USER_DATA` to read input and make dicission based on that i.e. to do `^C` or not.
